src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -38,13 +38,13 @@ public static class IdentifiableSecrets
 
     public static uint LongFormEncodedCommonAnnotatedKeySize => 88;
 
-    public static string CommonAnnotatedKeyCoreSignature = "JQQJ";
+    public static string CommonAnnotatedKeyCoreSignature => "JQQJ";
```

Comment:
Unrelated, but this shouldn't have been mutable.

Technically a breaking change.

Reply:
Unless this was added in the PR?